src/black/__init__.py

```diff
@@ -619,10 +619,9 @@ def path_empty(
     """
     Exit if there is no `src` provided for formatting
     """
-    if len(src) == 0:
-        if verbose or not quiet:
-            out(msg)
-            ctx.exit(0)
+    if not src and (verbose or not quiet):
```

Comment:
It's fine to use a single `if` statement here but keep the `len()` variant. The `src` type is specified as `Sized` so the only safe operation we can do on it is `__len__()`.

Reply:
I think we can leave this one. 

In case if an object doesn't have an explicit `__bool__` implementation but has `__len__` implemented, then `__len__` result will be used in order to convert an object into bool representation. 

Please correct me if I missed smth.

Reply:
Yeah, alright.